it/it-tests/src/test/java/it/analysis/SSLTest.java

```diff
@@ -0,0 +1,196 @@
+  private static final String CLIENT_TRUSTSTORE_PWD = "clienttruststorepwd";
+
+  @ClassRule
+  public static Orchestrator orchestrator = Category3Suite.ORCHESTRATOR;
+
+  private static Server server;
+  private static int httpsport;
+
+  @Before
+  public void deleteData() {
+    orchestrator.resetData();
+  }
+
+  public static void startSSLTransparentReverseProxy(boolean requireClientAuth) throws Exception {
+    int httpport = NetworkUtils.getNextAvailablePort();
+    httpsport = NetworkUtils.getNextAvailablePort();
+
+    // Setup Threadpool
+    QueuedThreadPool threadPool = new QueuedThreadPool();
+    threadPool.setMaxThreads(500);
+
+    server = new Server(threadPool);
+
+    // HTTP Configuration
+    HttpConfiguration http_config = new HttpConfiguration();
+    http_config.setSecureScheme("https");
+    http_config.setSecurePort(httpsport);
+    http_config.setSendServerVersion(true);
+    http_config.setSendDateHeader(false);
+
+    // Handler Structure
+    HandlerCollection handlers = new HandlerCollection();
+    handlers.setHandlers(new Handler[] {proxyHandler(), new DefaultHandler()});
+    server.setHandler(handlers);
+
+    ServerConnector http = new ServerConnector(server, new HttpConnectionFactory(http_config));
+    http.setPort(httpport);
+    server.addConnector(http);
+
+    Path serverKeystore = Paths.get(SSLTest.class.getResource("/analysis/SSLTest/serverkeystore.jks").toURI()).toAbsolutePath();
+    String keystorePassword = "serverkeystorepwd";
+    String serverKeyPassword = "serverp12pwd";
+    Path serverTruststore = Paths.get(SSLTest.class.getResource("/analysis/SSLTest/servertruststore.jks").toURI()).toAbsolutePath();
+    String truststorePassword = "servertruststorepwd";
+
+    // SSL Context Factory
+    SslContextFactory sslContextFactory = new SslContextFactory();
+    sslContextFactory.setKeyStorePath(serverKeystore.toString());
+    sslContextFactory.setKeyStorePassword(keystorePassword);
+    sslContextFactory.setKeyManagerPassword(serverKeyPassword);
+    sslContextFactory.setTrustStorePath(serverTruststore.toString());
+    sslContextFactory.setTrustStorePassword(truststorePassword);
+    sslContextFactory.setNeedClientAuth(requireClientAuth);
+    sslContextFactory.setExcludeCipherSuites("SSL_RSA_WITH_DES_CBC_SHA",
+      "SSL_DHE_RSA_WITH_DES_CBC_SHA", "SSL_DHE_DSS_WITH_DES_CBC_SHA",
+      "SSL_RSA_EXPORT_WITH_RC4_40_MD5",
+      "SSL_RSA_EXPORT_WITH_DES40_CBC_SHA",
+      "SSL_DHE_RSA_EXPORT_WITH_DES40_CBC_SHA",
+      "SSL_DHE_DSS_EXPORT_WITH_DES40_CBC_SHA");
+
+    // SSL HTTP Configuration
+    HttpConfiguration https_config = new HttpConfiguration(http_config);
+
+    // SSL Connector
+    ServerConnector sslConnector = new ServerConnector(server,
+      new SslConnectionFactory(sslContextFactory, HttpVersion.HTTP_1_1.asString()),
+      new HttpConnectionFactory(https_config));
+    sslConnector.setPort(httpsport);
```

Comment:
There are multiple naming conventions in this file `camelCase`, `snake_case`, and multiple lowercase words put together. It's hurting readability a bit. I suggest to rename to `httpsPort`, `httpPort`, `httpsConfig`, `serverTrustStore`, `keyStorePassword`, `trustStorePassword`. Admittedly this is not terribly important, do as you wish.